docs/model-dev-guide/model-management/checkpoints.rst

```diff
@@ -25,13 +25,13 @@ The Checkpoint Export API is a subset of the features found in the
  Querying Checkpoints
 **********************
 
-The :class:`~determined.experimental.client.ExperimentReference` class is a reference to an
-experiment. The reference contains the
-:meth:`~determined.experimental.client.ExperimentReference.top_checkpoint` method. Without
-arguments, the method will check the experiment configuration searcher field for the ``metric`` and
-``smaller_is_better`` values. These values are used to sort the experiment's checkpoints by
-validation performance. The searcher settings in the following snippet from an experiment
-configuration file will result in checkpoints being sorted by the loss metric in ascending order.
+The :class:`~determined.experimental.client.Experiment` class is a reference to an experiment.
+Within this class, there is a :meth:`~determined.experimental.client.Experiment.list_checkpoints`
+method. When called without arguments, this returns the experiment's checkpoints in sorted order
+using the ``metric`` and ``smaller_is_better`` values from the experiment configuration’s searcher
+field. These values are used to sort the experiment's checkpoints by validation performance. As an
+example, the searcher settings in the following experiment configuration file snippet will result in
+checkpoints being sorted by the loss metric in ascending order.
```

Comment:
Querying Checkpoints
***********************

Use the :class:`~determined.experimental.client.Experiment` class to reference an experiment. The :meth:`~determined.experimental.client.Experiment.list_checkpoints` method, when called without arguments, returns checkpoints sorted based on the ``metric`` and ``smaller_is_better`` values from the experiment configuration's searcher field. 

In the following experiment configuration file snippet, Determined will sort checkpoints by the loss metric and in ascending order.